pkg/service/operation/service.go

```diff
@@ -32,6 +32,30 @@ func (s Service) Status() framework.Status {
 	return framework.Status{Status: framework.StatusReady}
 }
 
+func (s Service) GetOperations(request GetOperationsRequest) (*GetOperationsResponse, error) {
+	ops, err := s.storage.GetOperations()
```

Comment:
could consider a cache on `s Service` in the future

Reply:
yes, definitely something we should think about